server/src/main/scala/controllers/TestRouter.scala

```diff
@@ -0,0 +1,44 @@
+package controllers
+
+import play.api.mvc._
+import play.api.routing.sird._
+import play.api.routing.{Router, SimpleRouter}
+import router.Routes
+import test.controllers.TestController
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class TestRouter @Inject()(cc: ControllerComponents, testController: TestController) extends SimpleRouter {
+
+  val router: Router = Router.from {
+
+    case GET(p"/test/hello") => testController.hello
+
+    case POST(p"/test/arrival") => testController.addArrival
+
+    case POST(p"/test/arrivals/$forDate") => testController.addArrivals(forDate)
+
+    case POST(p"/test/manifest") => testController.addManifest
+
+    case POST(p"/test/mock-roles") => testController.setMockRoles
+
+    case GET(p"/test/mock-roles-set") => testController.setMockRolesByQueryString
+
+    case DELETE(p"/test/data") => testController.deleteAllData
+
+  }
+
+
+  override def routes = router.routes
+
+}
+
+@Singleton
+class AppRouter @Inject()(defaultRoutes: Routes) extends SimpleRouter {
```

Comment:
I can't see this being used anywhere. In the main application.conf it's configured to use the CombinedRouter

If this one is for non-test mode then it should probably also not be in this file

Reply:
Non test don't need AppRouter . I will removing combinedRouter from application.conf